packages/protocol/tsconfig.lib.json

```diff
@@ -1,7 +1,9 @@
 {
     "extends": "../../tsconfig.lib.json",
     "compilerOptions": {
-        "outDir": "lib"
+        "outDir": "lib",
+        "target": "es2020",
+        "lib": ["es2020", "esnext"]
```

Comment:
should we bump package version because of this? cc @mroz22

Reply:
in package.json? nope, it will happen automatically during next connect release